### PR TITLE
kdenlive: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** latest

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
Since, there aren't a lot of commands here, how about adding version and help commands too.

**Comment by EmilySeville7cfg** [comment]
> Since, there aren't a lot of commands here, how about adding version and help commands too.

Makes sense. 👍🏻

File: pages/common/kdenlive.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> KDE's non-linear video editor.
+> More information: <https://manned.org/kdenlive>.
+
+- Start the video editor:
+
+`kdenlive`
+
+- Open a specific file:
+
+`kdenlive {{path/to/file}}`
```
Comment (pixelcmtd) [inline pages/common/kdenlive.md:12]: Does this open anything other than `.kdenlive` projects?

Comment (EmilySeville7cfg) [reply pages/common/kdenlive.md:12]: I don't know actually because I don't use `kdenlive`. 🤔

Comment (kbdharun) [reply pages/common/kdenlive.md:12]: Yeah, I checked it. It opens only kdenlive project files.

Comment (kbdharun) [inline pages/common/kdenlive.md:12]:
```suggestion
`kdenlive {{path/to/file}}.kdenlive`
```

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**